### ISSUE TITLE
[REFACTOR] :recycle: Migration to Nextjs for Landing in a different repo.

### ISSUE THREAD

**Issue by is-it-ayush** [MultiEmail/frontend#106]
### Description

Here's a suggestion, 

### How? 
We can transfer the landing page to a different repo and refactor it to Nextjs (which won't be much of an issue as it's still in it's really early days) while leaving the rest as is.

### Why? 
User's don't actually need the landing page with the main mailing app combined. The one's who want to run the app can separately download the latest `build` from release's for the client and the server whereas landing page is something we need for `multimail.us` domain not for the whole mailing app itself.

### Integration:
Later we can, assign the app on the `app` subdomain such that the main URL become's `app.multiemail.us` for the app itself & The landing page will be available via `multiemail.us`.

### Proposed Arch: 
![Untitled-2022-10-15-1834](https://user-images.githubusercontent.com/36449128/196052626-7540ecea-28d6-4d51-ad07-af424a23b133.png)

Feedback appreciated.

> tldr; Seperate out Landing Page's from Mailing App. Refactor it to NextJs in a different repo. The main mailing app can stay here and be worked on. Reason's for such are SEO and distinction.

**Comment by aayushchugh** [comment]
@is-it-ayush I agree

**Comment by is-it-ayush** [comment]
Thank You! Currently #86 and #81 are related to the Landing Page. Once they're merged. I'll work on this and migrate the landing to NextJS. Further Landing Page PR's and issue's related to Landing would be dealt in it's own repo. Hence, all PR's here related to landing page would be held after the two issues are closed.
